static/src/javascripts/projects/common/modules/commercial/front-commercial-components.js

```diff
@@ -27,17 +28,17 @@ define([
                 containerIndex = config.page.contentType === 'Network Front' ? 3 : 2;
             }
 
-            return $adSlotWrapper
-                .append($adSlot)
-                .insertAfter($containers[containerIndex]);
-        }
+            $adSlotWrapper.append($adSlot);
```

Comment:
It's not attached to the dom at that point, so doesn't participate in layout or painting.